lib/commands/general.js

```diff
@@ -102,7 +102,7 @@ commands.installApp = async function (appPath) {
 
   let {apkPackage} = await this.adb.packageAndLaunchActivityFromManifest(appPath);
   let opts = {
-    localApkPath: appPath,
+    app: appPath,
     pkg: apkPackage,
```

Comment:
This looks like it is looking for `appPackage`, not `pkg`.

https://github.com/appium/appium-android-driver/blob/c5c8c1bb0b855283c83c760e41a91ca03b003204/lib/android-helpers.js#L261